src/setupTest.js

```diff
@@ -49,5 +49,21 @@ mergeConfig({
   PREVIEW_BASE_URL: process.env.PREVIEW_BASE_URL || '',
 }, 'CourseAuthoringConfig');
 
+class ResizeObserver {
+  observe() {
+    // do nothing
+  }
+
+  unobserve() {
+    // do nothing
+  }
+
+  disconnect() {
+    // do nothing
+  }
+}
+
+window.ResizeObserver = ResizeObserver;
```

Comment:
I was getting "ResizeObserver is not defined" errors. I decided to see how paragon is handling those and I found https://github.com/openedx/paragon/blob/7e4a81f9f2350c54362baa852c75dc32c2c2694e/src/setupTest.js#L7-L21, so I copied that here.